content/pt/docs/6.internals-glossary/1.context.md

```diff
@@ -27,137 +27,137 @@ function (context) {
     error,
     $config
   } = context
-  // Server-side
+  // No Lado do Servidor
   if (process.server) {
     const { req, res, beforeNuxtRender } = context
   }
-  // Client-side
+  // No Lado do Cliente
   if (process.client) {
     const { from, nuxtState } = context
   }
 }
 ```
 
-## Universal keys
+## Chaves universais
 
-These keys are available both on client-side and server-side.
+Essas chaves estão disponíveis em ambos lado do cliente e servidor.
 
 ### app
 
 `app` (_NuxtAppOptions_)
 
-The root Vue instance options that includes all your plugins. For example, when using `i18n`, you can get access to `$i18n` through `context.app.i18n`.
+As opções da instância raiz do Vue que incluem todos os seus plugins. Por exemplo, quando estiver usando o `i18n`, você pode obter o acesso ao `$i18n` através do `context.app.i18n`.
 
 ### store
 
-`store` ([_Vuex Store_](https://vuex.vuejs.org/api/#vuex-store-instance-properties))
+`store` ([_Memória do Vuex_](https://vuex.vuejs.org/api/#vuex-store-instance-properties))
 
-Vuex Store instance. **Available only if the [vuex store](/docs/directory-structure/store) is set**.
+Instância da Memória do Vuex. **Apenas disponível se a [memória do vuex](/docs/directory-structure/store) estiver definida**.
 
 ### route
 
-`route` ([_Vue Router Route_](https://router.vuejs.org/api/#the-route-object))
+`route` ([_Rota do Roteador do Vue_](https://router.vuejs.org/api/#the-route-object))
 
-Vue Router route instance.
+Instância da Rota do Roteador do Vue.
 
 ### params
 
 `params` (_Object_)
 
-Alias of `route.params`.
+Pseudónimo de `route.params`.
 
 ### query
 
 `query` (_Object_)
 
-Alias of `route.query`.
+Pseudónimo de `route.query`.
 
 ### env
 
 `env` (_Object_)
 
-Environment variables set in `nuxt.config.js`, see [env API](/docs/configuration-glossary/configuration-env).
+Variáveis de ambiente definidas dentro do ficheiro `nuxt.config.js`, consulte a [API de env](/docs/configuration-glossary/configuration-env).
 
 ### isDev
 
 `isDev` (_Boolean_)
 
-Boolean to let you know if you're in dev mode, can be useful for caching some data in production.
+Booleano para permitir você saber se está em modo de desenvolvimento, pode ser útil para cacheamento de algum dado em produção.
 
 ### isHMR
 
 `isHMR` (_Boolean_)
 
-Boolean to let you know if the method/middleware is called from webpack hot module replacement (_true only on client-side in dev mode_).
+Booleano que permitir você saber se o método/intermediário é chamado a partir da substituição instantânea de módulo do webpack (_é `true` apenas no lado do cliente no modo de desenvolvimento_).
```

Comment:
`Booleano que permite saber...`

Reply:
@ChristopheCBV I almost can understand the reason, but do you think the translation was too literal? 😅